### PR TITLE
Add Azure Scale Set support mention in Docs

### PR DESCRIPTION
**Pull request by avaakash**
Signed-off-by: Akash Shrivastava <akash@chaosnative.com>

## Proposed changes
Adds SCALE_SET env in the docs 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: mkdocs/docs/experiments/categories/azure/azure-instance-stop.md [hunk 1]
```diff
@@ -195,6 +200,44 @@ spec:
...
+          VALUE: '60'
+```
+
+### Scale Set Instances
+
+It contains comma separated list of instance names subjected to instance stop chaos belonging to Scale Set or AKS. It can be tuned via `SCALE_SET` ENV.
+
+Use the following example to tune this:
+
+[embedmd]:# (https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-instance.yaml yaml)
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/azure/azure-instance-stop.md:216]: Can we embed this file instead: https://github.com/litmuschaos/litmus/blob/a0ac61ed1507fea458e8d73e9bee04578a86f06e/mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-scale-set-instance.yaml?

File: mkdocs/docs/experiments/categories/azure/azure-instance-stop/chaos-interval.yaml [hunk 2]
```diff
@@ -22,3 +22,6 @@ spec:
           value: 'instance-01,instance-02'
         - name: RESOURCE_GROUP
           value: '<resource group of AZURE_INSTANCE_NAME>'
+        # accepts enable/disable value. default is disable
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/azure/azure-instance-stop/chaos-interval.yaml:25]: It is not a mandatory field so we will provide it in https://github.com/litmuschaos/litmus/blob/a0ac61ed1507fea458e8d73e9bee04578a86f06e/mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-scale-set-instance.yaml file only

Comment (avaakash) [reply mkdocs/docs/experiments/categories/azure/azure-instance-stop/chaos-interval.yaml:25]: Should I mention it in the Optional Fields section?

File: mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-scale-set-instance.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: azure-instance-stop-sa
+  experiments:
+  - name: azure-instance-stop
+    spec:
+      components:
+        env:
+        # comma separated list of azore instance names
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/azure/azure-instance-stop/azure-scale-set-instance.yaml:15]:
```suggestion
        # comma separated list of azure instance names
```

File: mkdocs/docs/experiments/categories/azure/azure-instance-stop/chaos-interval.yaml (hunk 2 above)
Comment (ispeakc0de) [reply mkdocs/docs/experiments/categories/azure/azure-instance-stop/chaos-interval.yaml:25]: I mean we can remove SCALE_SET ENV from here and from other files except the `azure-scale-set-instance-yaml` as it is an optional field

File: mkdocs/docs/experiments/categories/azure/azure-instance-stop.md [hunk 4]
```diff
@@ -122,7 +122,7 @@
       <tr> 
         <td> AZURE_INSTANCE_NAME </td>
         <td> Instance name of the target azure instance</td>
-        <td> </td>
+        <td> For AKS ndoes, the instance name is from the scale set section in Azure and not the node name from AKS node pool </td>
```
Comment (ispeakc0de) [inline mkdocs/docs/experiments/categories/azure/azure-instance-stop.md:125]:
```suggestion
        <td> For AKS nodes, the instance name is from the scale set section in Azure and not the node name from AKS node pool </td>
```

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**